bucket/cv4pve-pepper.json

```diff
@@ -0,0 +1,29 @@
+{
+    "version": "1.5.0",
+    "description": "Launching SPICE remote-viewer to access VM running on Proxmox VE.",
+    "homepage": "https://github.com/Corsinvest/cv4pve-pepper",
+    "license": "GPL-3.0-only",
+    "depends": "extras/virt-viewer",
```

Comment:
Could you share a link to this virt-viewer manifest from extras with me?

Reply:
It should also be suggest

Reply:
The virt-viewer manifest is in this pull request: https://github.com/ScoopInstaller/Extras/pull/6451.